_extensions/custom_roles.py

```diff
@@ -8,6 +8,12 @@
 
 
 def setup(app):
+    # role to link to cpython files
+    app.add_role(
+        "cpy-file",
+        autolink("https://github.com/python/cpython/blob/main/%s"),
```

Comment:
```suggestion
        autolink("https://github.com/python/cpython/blob/main/{}"),
```

Corresponding change here